### ISSUE TITLE
Missing DLP Tier 1 support in 7MS/Mozilla/BodgeIt themes

### ISSUE THREAD

**Issue by bkimminich** [juice-shop/juice-shop#841]
In `v8.4.0` the following packaged customizations would become invalid due to missing declaration of `keywordsForPastebinDataLeakChallenge` on one of their products:
* [x] `7ms.yml`
* [x] `bodgeit.yml`
* [x] `mozilla.yml`

> :information_source: Read #792 carefully for details about the puzzle aspect and how the challenge is supposed to work before submitting any PR or creating any PasteBin entries!

## How to fix this issue
* One product in each config must be chosen as the _Pastebin Leak Challenge Product_ and receive `keywordsForPastebinDataLeakChallenge` property with a list of expected trigger words
* An anonymous PasteBin paste must be created for each config that **makes sense in the "back story" of the chosen unsafe product** and contains the trigger words (including a little puzzle to find them)
* The `internetResourcesSpec.js` placeholder `xit`'s must be filled for each new PasteBin paste and be activated

**Comment by supra08** [comment]
@bkimminich, weren't we using two products for the validation of the Pastebin Challenge? And I believe we can put a section in the docs the prerequisites for setting up this challenge as it would vary from installation to installation.

**Comment by bkimminich** [comment]
No, only one product, but we chose two "ingredients" as a danger-combo and the puzzle was to identify them in your pasted data.

It will be documented for custom configs how this challenge needs to be prepared, but as the Juice Shop delivers 3 extra sample configs out of the box, those must be complete and all challenges should work. That's why I opened this reminder-issue to address it before 8.4.0 goes live.
